lib/readline.js

```diff
@@ -72,9 +70,12 @@ function Interface(input, output, completer, terminal) {
   this.historySize = historySize;
 
   // Check arity, 2 - for async, 1 for sync
-  this.completer = completer.length === 2 ? completer : function(v, callback) {
-    callback(null, completer(v));
-  };
+  if (typeof completer === 'function') {
+    this.completer = completer.length === 2 ? 
```

Comment:
Sure, sorry. Since the line was 79 characters before and the indentation adds 2, it is now 81. If you change the `===` to `==`, we can keep it at 80 and remove a line. Like this:

```diff
-  this.completer = completer.length === 2 ? completer : function(v, callback) {
-    callback(null, completer(v));
-  };
+  if (typeof completer === 'function') {
+    this.completer = completer.length == 2 ? completer : function(v, callback) {
+      callback(null, completer(v));
+    };
+  }
```

The `===` and `==` change shouldn't make a difference, since `.length` will always be a number.